app/conversation.js

```diff
@@ -1,145 +1,156 @@
 
 import React, { Component, } from 'react';
-import { AppRegistry, Text,View,StyleSheet,Image,TouchableHighlight,TouchableOpacity, ScrollView} from 'react-native';
+import { AppRegistry, 
+  Text,
+  View,
+  StyleSheet,Image,TouchableHighlight,
+  TouchableOpacity, ScrollView, AsyncStorage } from 'react-native';
 
-import { Actions } from 'react-native-router-flux';
-import { GiftedChat,Bubble } from 'react-native-gifted-chat';
+import GliaBubble from './gliaBubble.js';
+import PubNub from 'pubnub';
 
-import SendBird from 'sendbird';
-var sb = null;
+import { GiftedChat,Bubble } from 'react-native-gifted-chat';
 
+const pubnub = new PubNub({
+    subscribeKey: "sub-c-db3b3db0-2d30-11e7-87b6-02ee2ddab7fe",
+    publishKey: "pub-c-db8bb359-b20e-415f-80d9-0958db03625c",
+    ssl: true,
+  });
 
 export default class Conversation extends Component {
+  
   constructor(props) {
     super(props);
+
     this.state = {
       messages: [],
-      channel: null,
-      isConnected: false,
-      chanSize: 0,
+      userID: "",
+      username: "",
       channelTitle: "",
-      };
-    this.onSend = this.onSend.bind(this);
-
-
-
-    //Do things with channels
-
+      channNumber: 1,
+      channelID: "Conversation 1",
+      userThumbnail: "https://www.timeshighereducation.com/sites/default/files/byline_photos/default-avatar.png",
```

Comment:
[http://adityaaggarwal.com/silver_maple.png](Profile Image) The profile image above works for test cases but later down the line it would be great to use images randomly generated that looks like the link added here